Source/WebCore/rendering/RenderThemeMac.mm

```diff
@@ -2661,6 +2661,19 @@ static void paintAttachmentIconBackground(const RenderAttachment& attachment, Gr
     }
 }
 
+static bool shouldDrawIcon(const String& title)
+{
+    // iWork file icons will be generic due to sandboxing, we should not paint
+    // them. The thumbnail will be painted once it is fetched in the UIProcess.
```

Comment:
Very minor nit — we generally avoid discussing the process model inside WebCore code. Maybe we could rephrase this comment as "The thumbnail will be painted by the client."?

Reply:
Good point - I will re-phrase.